CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * UX consistency: Use Save & close button label stripes-component translation key. Refs UIU-3078.
 * Fix two "triangle down" icons in select element of "Copy existing fee/fine owner table entries" modal. Refs UIU-2929.
 * Fix incorrect translation key having count-disagreements in Pay fees/fines modal in Fees/Fines Page. Refs - UIU-1097.
+* Trim input values and convert empty string to null. Refs - UIU-2049.
```

Comment:
This is not accurate. `{ foo: "bar", bat: null }` is different than `{ foo: bar }`. You're not setting empty strings to null; when you find a value that is an empty string, you are deleting that property altogether.